backend/dal/Services/Concrete/ProjectService.cs

```diff
@@ -799,7 +841,28 @@ public async System.Threading.Tasks.Task<Project> SetStatusAsync(Project project
             originalProject.Status = toStatus;
             originalProject.Metadata = project.Metadata;
             project.CopyRowVersionTo(originalProject);
-            this.Context.CommitTransaction();
+            try
```

Comment:
Regrettably incorrect. Same as above comment.

Reply:
Gotchya, that makes sense. I originally thought it was a frontend error but read the docs the error message pointed to and then changed my mind and went with that approach. Adjusting now!